docs/man_pages/project/testing/test-android.md

```diff
@@ -0,0 +1,22 @@
+test android
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ tns test android [--device <Device ID>] [--watch]`
+
+Runs the tests in your project on connected Android devices and emulators. 
+
+### Options
+* `--watch` - If set, when you save changes to the project, changes are automatically synchronized to the connected device and tests are re-run.
```

Comment:
Shouldn't the watch option be listed as second because it is second in the usage section?